sdk/storage/test-resources.json

```diff
@@ -58,7 +58,7 @@
             "signedPermission": "rwdlacup",
             "signedResourceTypes": "sco",
             "keyToSign": "key2",
-            "signedExpiry": "2022-01-01T23:59:00Z"
+            "signedExpiry": "2023-01-01T23:59:00Z"
```

Comment:
How about just 2099? so that we don't have to fix it every year

Reply:
not sure if that would be valid.. Usually the Azure Portal lets to set 1 year the most..  but let's try it

Reply:
let's use [this PR](https://github.com/Azure/azure-sdk-for-cpp/pull/3210) in case 2099 works

Reply:
> not sure if that would be valid.. Usually the Azure Portal lets to set 1 year the most.. but let's try it

I tried on Azure Portal. I can generate a SAS token expires in 2099.

Reply:
beautiful!  I have updated to be 99 . Thank you